tools/docker-network/config.docker.json

```diff
@@ -70,8 +73,12 @@
     "disablePlugins": "portcheck",
     "enablePlugins": []
   },
+  "scheduler": {
+    "maxBufferSize": 150,
+    "rate": "20ms"
+  },
   "pow": {
-    "difficulty": 2,
+    "difficulty": 0,
```

Comment:
Why these changes and not the default values?

Reply:
That's a badly commited file that I changed for local debugging and testing. Reverted the changes already.